docker-compose.arm64.yml

```diff
@@ -0,0 +1,5 @@
+
+
+services:
+  postgres:
+    image: duvel/postgis:12-2.5-arm64
```

Comment:
Any reason to prefer such an ancient version?

Reply:
ARM images are not available yet, but here is some fresh update
https://github.com/postgis/docker-postgis/issues/216#issuecomment-1763399631